content/en/dora_metrics/failures/_index.md

```diff
@@ -35,13 +37,149 @@ Failed deployments events, currently interpreted through incident events, are us
 
 ## Selecting an incident data source
 
-{{< whatsnext desc="DORA Metrics supports the following data sources for deployment events. See the respective documentation to set up a data source for your deployment events:" >}}
-  {{< nextlink href="/dora_metrics/failures/pagerduty" >}}PagerDuty{{< /nextlink >}}
-  {{< nextlink href="/dora_metrics/failures/incident_api" >}}Incident Event API{{< /nextlink >}}
-{{< /whatsnext >}}
+{{< tabs >}}
+{{% tab "PagerDuty" %}}
```

Comment:
Because only PagerDuty has a tab, I would put it in the `## Configuring an incident data source` header, below the setup instructions